tools/clang/include/clang/SPIRV/SpirvInstruction.h

```diff
@@ -2036,6 +2146,24 @@ class SpirvDebugLexicalBlock : public SpirvDebugInstruction {
   SpirvDebugInstruction *parent;
 };
 
+/// Represent DebugScope. We assume that DXC does not generate inlining
+/// information. We do not add "Inlined" operand.
```

Comment:
What does this mean?   How does this Inlined operand relate to DebugScope?  I expect this would be generated as a frame descriptor when the inliner runs.  But if DXC does not inline, then I'm not sure how this affects `SpirvDebugScope`.

Reply:
`DebugScope` has two operands other than result id, type id, and extension:
- result id of an instruction for a lexical scope
- result id of `DebugInlinedAt` which contains inlining information, which means `DebugScope` is used to specify "a set of instructions shown below is inlined from XXX function".

However, inlining will be done by spirv-opt, not DXC. We do not need the operand for result id of `DebugInlinedAt`. I tried to explain this situation but it seems my comment was too short. Let me explain it in detail.